myapp/db/seeds.rb

```diff
@@ -1,7 +1 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+User.create(username: 'First User', password: '12345')
```

Comment:
nitpick: `username` should be an string without space as ideal practice.

Reply:
Fixed it and added validation in User model.
Please refer e430cc9